app/javascript/dashboard/store/modules/campaigns.js

```diff
@@ -42,6 +42,17 @@ export const actions = {
       commit(types.SET_CAMPAIGN_UI_FLAG, { isCreating: false });
     }
   },
+  update: async function updateCampaign({ commit }, { id, ...updateObj }) {
```

Comment:
Can we change it like below to avoid multiple names for same function?
`async update(...) {`